### DISCUSSION TITLE
How can I implement like that with JointJS ! Thank you

### DISCUSSION THREAD

**Discussion by quyphuc2111** [clientIO/joint#2415] [Q&A]
### Introduction
I'm trying to create node center, when i focus node ( highlight and create sub node ). 
Help me with related instructions about it.
Thank you!!
![image](https://github.com/clientIO/joint/assets/58882661/e7a86096-8ed1-45ea-abdf-2f18917db510)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
> I'm trying to create node center, when i focus node ( highlight and create sub node ).

Sorry I don't know what you mean by that. Could you elaborate please?

There are plenty of examples showing how to highlight an element: https://www.jointjs.com/demos?feature=Highlighters

**Reply by quyphuc2111** [reply]
https://www.awesomescreenshot.com/video/23075363?key=5e5871fc1f994e550ea58c8fc1a8d59b

On the Mindmanager site , I want to do the same thing. My English is a bit poor so I can't describe it more clearly

**Reply by kumilingus** [reply]
If you want to implement a mind map, we do have an example application in **JointJS plus**: https://www.jointjs.com/demos/mindmap
